Formula/hostsfile.rb

```diff
@@ -0,0 +1,25 @@
+class Hostsfile < Formula
+  desc "CLI for manipulating /etc/hosts files"
+  homepage "https://github.com/kevinburke/hostsfile"
+
+  url "https://github.com/kevinburke/hostsfile/archive/1.4.tar.gz"
+  sha256 "6702d19c45d0f3474d3a528cff7da67ad6e7fb912e027c910b1887961a24ecb2"
+  license "MIT"
+  head "https://github.com/kevinburke/hostsfile"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
This is deprecated and shouldn't be used anymore.

Reply:
IIRC it's still necessary if the project doesn't have a go.mod, and imports other packages.